packages/api/src/command/medical/address/generate-geocoded-addresses.ts

```diff
@@ -0,0 +1,36 @@
+import { Address } from "../../../domain/medical/address";
+import { AddressGeocodingResult, geocodeAddress } from "../../../external/aws/address";
+
+/**
+ * Generates geo coordinates for addresses that don't already have them. Returns these addresses along with their relevance score from geocoding.
+ *
+ * @param addressList a list of Address objects.
+ * @returns a list of Address objects that got updated with coordinates, relevance scores, and suggested labels.
+ */
+export async function generateGeocodedAddresses(
```

Comment:
nit: the original name seemed more representative of what this does? `addGeographicCoordinates`

`generateGeocodedAddresses` doesn't really generate addresses :)